README.md

```diff
@@ -21,7 +21,9 @@ If you would like to be notified of future conferences that would be added to th
 | [LaravelLive India](https://laravellive.in/) | 19 Mar | New Delhi | A PHP Laravel Conference. |
 | [Banglore Container Conference](http://www.containerconf.in/) | 7 Apr | Park Plaza (5 star), ORR, Bengaluru | Bengaluru Container Conference 2017 (BCC ’17) is the first conference on container technologies in India. The main focus of this conference is “Containers in Production”. |
 | [GIDS](http://www.developermarch.com/developersummit/) | 25-28 April | J. N. Tata Auditorium, Bengaluru | With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.  |
+| [Oracle Code New Delhi](https://developer.oracle.com/code/newdelhi) | 09-10 May | Pragati Maidan New Delhi | Oracle Code is a free event for developers to learn about the latest developer technologies, practices, and trends.Learn from technical experts, industry leaders, and other developers in keynotes, sessions, and hands-on labs. Experience cloud development technology in the Code Lounge with workshops and other live, interactive experiences and demos. |
 | [RootConf](https://rootconf.in/2017/) | 11-12 May | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge about building resilient and scalable systems. |
+| [Amazon AWS Summit](https://aws.amazon.com/summits/new_delhi/) | 12 May | JW Marriott Hotel Aerocity New Delhi | The AWS Summit brings together the cloud computing community to connect, collaborate and learn about AWS. In this action-packed, one-day event, you can choose from over 25 breakout sessions covering a broad range of topics and technical depth. Attend introductory sessions on new and existing AWS services or participate in sessions that go deep into architecture and code. All sessions will be delivered by subject matter experts from AWS, Partners or expert customers who can share their real world experiences and lessons learned.|
```

Comment:
Can you limit the description till just the first sentence? I think it conveys enough information.

I don't want the table to bloat up with huge descriptions. :smile: (though I agree I have some cleaning up to do wrt other conferences with big descriptions)